KerbalStuff/blueprints/api.py

```diff
@@ -103,10 +103,19 @@ def version_info(mod: Mod, version: ModVersion) -> Dict[str, str]:
         "download_path": url_for('mods.download', mod_id=mod.id,
                                  mod_name=mod.name,
                                  version=version.friendly_version),
-        "changelog": version.changelog
+        "changelog": version.changelog,
+        "downloads": str(mod_version_download_count(mod, version)),
```

Comment:
Why return a string and not an int?

Reply:
Fair point, it was because the function's return type was `Dict[str, str]`, but that can be changed. Done.